<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuCitas</class>
 <widget class="QMainWindow" name="MenuCitas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="menuDates">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>MenuCitas.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Back">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
back-groud color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ir al men√∫ principal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddProducto">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton{

	background-color: rgb(255, 255, 255) ;
	
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}

QPushButton:hover{

	background-color:  rgb(85, 170, 0);
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_consultProdcto">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color: rgb(255, 255, 255) ;
	
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}

QPushButton:hover{

	background-color: rgb(255, 170, 0);
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}</string>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeleteProducto">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color: rgb(255, 255, 255) ;
	
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}

QPushButton:hover{

	background-color:  rgb(255, 0, 0);
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
